name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/INGDiBaPDFExtractor.java

```diff
@@ -320,6 +325,11 @@ private void addAdvanceTaxTransaction()
                 .match("^Valuta (?<date>[\\d]{2}\\.[\\d]{2}\\.[\\d]{4})$")
                 .assign((t, v) -> t.setDateTime(asDate(v.get("date"))))
 
+                // Ex-Dividenden-Tag 04.01.2021
```

Comment:
// Ex-Dividenden-Tag 04.01.2021 
not the same for regEx
^Ex-Tag (?<exDate>\\d+.\\d+.\\d{4})$

Reply:
changed & done locally